runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -232,6 +232,7 @@
 	ITEM_BOTTOMLESS_COMPOST_BUCKET(BOTTOMLESS_COMPOST_BUCKET, BOTTOMLESS_COMPOST_BUCKET_22997),
 	ITEM_BASILISK_JAW(BASILISK_JAW, NEITIZNOT_FACEGUARD),
 	ITEM_HELM_OF_NEITIZNOT(HELM_OF_NEITIZNOT, NEITIZNOT_FACEGUARD),
+	ITEM_TWISTED_HORNS(TWISTED_HORNS, SLAYER_HELMET, SLAYER_HELMET_I),
```

Comment:
Pretty sure the normal slayer helmet doesn't have twisted horns :thinking:

Reply:
I was going according to https://oldschool.runescape.wiki/w/Twisted_horns
is there a better site for more accurate information?

Reply:
No, the mapping is supposed to make the Twisted Slayer Helmets to the Horns

Reply:
oh I understand now. Thanks for your patience while I am learning.